crates/bevy_ecs/src/schedule/executor/single_threaded.rs

```diff
@@ -44,13 +61,57 @@ impl SystemExecutor for SingleThreadedExecutor {
     }
 
     fn run(&mut self, schedule: &mut SystemSchedule, world: &mut World) {
+        /// return the next system not exempt from stepping starting at, and
```

Comment:
These docs need to be changed to reflect this method's use when stepping is disabled as well.